### PR TITLE
ISPN-11070 Use REST Subject to perform cache admin ops

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-11070

### CODE REVIEW

File: core/src/main/java/org/infinispan/manager/EmbeddedCacheManagerAdmin.java [hunk 1]
```diff
@@ -64,4 +66,11 @@
     * @return the cache
     */
    <K, V> Cache<K, V> getOrCreateCache(String name, Configuration configuration);
+
+   /**
+    *
+    * @param subject
+    * @return
+    */
```
Comment (ryanemerson) [inline core/src/main/java/org/infinispan/manager/EmbeddedCacheManagerAdmin.java]: Javadoc needs completing

Comment (tristantarrant) [reply core/src/main/java/org/infinispan/manager/EmbeddedCacheManagerAdmin.java]: Done.

**Comment by ryanemerson** [comment]
Unused imports in `EmbeddedCacheManagerAdmin` :cry:

**Comment by tristantarrant** [comment]
Fixed

**Comment by ryanemerson** [comment]
Thanks @tristantarrant

**Pull request merged**